src/components/Authors/Authors.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {Fragment} from 'react';
+import block from 'bem-cn-lite';
+import {Author} from '../../models';
+
+import './Authors.scss';
+
+const b = block('authors');
+
+const MAX_NUMBER_DISPLAYED_AUTHORS = 3;
+
+export interface AuthorsProps {
+    title: string;
+    usersMetadata: string;
+}
+
+export const Authors: React.FC<AuthorsProps> = (props) => {
+    const {title, usersMetadata} = props;
+
+    const users = JSON.parse(usersMetadata.replace(/'/g, '"'));
+
+    if (!users || Object.keys(users).length === 0) {
+        return null;
+    }
+
+    const avatars = getAvatars(users);
+
+    return (
+        <div className={b()}>
+            <div className={b('title')}>{title}</div>
+            <div className={b('avatars')}>{avatars}</div>
+        </div>
+    );
+};
+
+function getAvatars(authors: Author[]) {
+    if (authors.length === 1) {
+        return getOneAvatar(authors[0]);
+    }
+
+    const displayedAvatars: JSX.Element[] = [];
+
+    authors.forEach((author: Author, id: number) => {
+        if (id < MAX_NUMBER_DISPLAYED_AUTHORS) {
+            displayedAvatars.push(getAvatar(author));
+        }
+    });
```

Comment:
Может тоже сначала слайсом отрезать первые `MAX_NUMBER_DISPLAYED_AUTHORS`?

и `id` выглядит странно

```suggestion
    authors.forEach((author: Author, index: number) => {
        if (index < MAX_NUMBER_DISPLAYED_AUTHORS) {
            displayedAvatars.push(getAvatar(author));
        }
    });
```